Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorComponentModeBus.h

```diff
@@ -142,6 +142,10 @@ namespace AzToolsFramework
             /// has the same Component on it, move all Components into ComponentMode.
             virtual void AddSelectedComponentModesOfType(const AZ::Uuid& componentType) = 0;
 
+            // If the user changes component edit mode through the switcher, switch the 
```

Comment:
update: In code it's fine to just call it Component Mode (not Component Edit Mode) to be more consistent